.env.example

```diff
@@ -1,9 +1,10 @@
-DB_USERNAME=postgres
-DB_PASSWORD=db_password
-DB_NAME=barefoot_nomad
-DB_HOST=127.0.0.1
-DB_TEST_NAME=barefoot_test
-PORT = 
+DB_USERNAME=database username
```

Comment:
Why do you duplicate DB_USERNAME? and so on

Reply:
This is a screenshot of my  `.env.example` file. There is no duplication 
![Screenshot (34)](https://user-images.githubusercontent.com/46751363/64114625-61ffa880-cd85-11e9-8649-d7de7dc0e4df.png)